.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: ruby
 rvm:
-  - 2.5
-  - 2.6
+  - 2.6.5
```

Comment:
Can we drop `patch` version? I am not sure it's worth the effort update it every time a new `patch` version is released. There can be no backwards-incompatible changes in both `minor` or `patch` version according to semver.

Reply:
Sure can do, no problem.